cg/store/crud/read.py

```diff
@@ -1031,20 +1051,8 @@ def cases_to_analyse(self, workflow: Workflow = None, limit: int = None) -> list
             filter_functions=case_filter_functions,
             workflow=workflow,
         )
-
-        families: list[Query] = list(cases.order_by(Case.ordered_at))
-        families = [
-            case_obj
-            for case_obj in families
-            if case_obj.latest_sequenced
-            and (
-                case_obj.action == CaseActions.ANALYZE
-                or not case_obj.latest_analyzed
-                or case_obj.latest_analyzed < case_obj.latest_sequenced
```

Comment:
With the removal of this part of the code, we can deprecate the Case property `latest_sequenced`, if you consider it under the scope of this PR. Ignore otherwise

Reply:
Good catch! Another PR I think!